### PR TITLE
Add calculate.js and operate.js file

### PR DESCRIPTION
**Pull request by lazyCode-404**
### The following were implemented:
- Created a logic directory.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.

- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,29 +1,54 @@
...
+import react from 'react';
+import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { calc: ['AC', '+/-', '%', '/', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+'] };
+class Calculator extends react.Component {
+  constructor() {
+    super();
```
Comment (cynthiainga) [inline src/components/Calculator.js]: - [ ] Please `props` need to get passed to child components of the Calculate.js component, kindly pass it 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @lazyCode-404, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Correct gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @lazyCode-404, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lazyCode-404** [comment]
Thank you.

**Pull request merged**